File: web/src/model/victron_controller/dashboard/from_0_2_0_actual-i32.ts
// @ts-nocheck
import {ActualI32 as v0_2_0_ActualI32} from './v0_2_0/ActualI32'
import {ActualI32 as dashboard_ActualI32} from './ActualI32'

export function convert__actual_i32__from__0_2_0(from: v0_2_0_ActualI32): dashboard_ActualI32 {
    return new dashboard_ActualI32 (
        from.value,
        JSON.parse(JSON.stringify(from.freshness)),
        from.since_epoch_ms
    )
}

File: crates/dashboard-model/src/victron_controller/dashboard/from_0_2_0_actual_i32.rs


pub fn convert__actual_i32__from__0_2_0(from: &crate::victron_controller::dashboard::v0_2_0::actual_i32::ActualI32) -> crate::victron_controller::dashboard::actual_i32::ActualI32 {
    crate::victron_controller::dashboard::actual_i32::ActualI32 {
        value: from.value.clone(),
        freshness: serde_json::from_value(serde_json::to_value(&from.freshness).unwrap()).unwrap(),
        since_epoch_ms: from.since_epoch_ms.clone(),
    }
}